docker-compose.test.yml

```diff
@@ -2,49 +2,76 @@ version: "3"
 
 services:
   auth:
-    image: openslides-auth-test
+    image: openslides-auth-dev
     restart: always
     depends_on:
+      - datastore-reader
+      - datastore-writer
       - message_bus
       - cache
+    env_file: services.env
     environment:
-      - CACHE_HOST=cache
-      - CACHE_PORT=6379
-      - MESSAGE_BUS_HOST=message_bus
       - CREATE_KEYS=1
+      - TEST=1
     volumes:
       - ./auth/src:/app/src
     ports:
       - "9004:9004"
-    networks:
-      - auth
 
   message_bus:
     image: redis:alpine
-    networks:
-      - auth
 
   cache:
     image: redis:alpine
     expose:
       - 6379
-    networks:
-      - auth
+
+  datastore-reader:
+    build:
+      context: "https://github.com/OpenSlides/openslides-datastore-service.git"
+      dockerfile: Dockerfile.prod
+      args:
+        MODULE: "reader"
+        PORT: "9010"
+    image: openslides-datastore-reader
+    ports:
+      - "9010:9010"
+    depends_on:
+      - postgresql
+  datastore-writer:
+    build:
+      context: "https://github.com/OpenSlides/openslides-datastore-service.git"
+      dockerfile: Dockerfile.prod
+      args:
+        MODULE: "writer"
+        PORT: "9011"
+    image: openslides-datastore-writer
+    ports:
+      - "9011:9011"
+    environment:
+      - DATASTORE_ENABLE_DEV_ENVIRONMENT=1
+      - COMMAND=create_initial_data
+      - DATASTORE_INITIAL_DATA_FILE=https://raw.githubusercontent.com/OpenSlides/OpenSlides/openslides4-dev/docs/example-data.json
+    depends_on:
+      - postgresql
+      - redis
+
+  postgresql:
+    image: postgres:11
+    environment:
+      - POSTGRES_USER=openslides
+      - POSTGRES_PASSWORD=openslides
+      - POSTGRES_DB=openslides
+  redis:
+    image: redis:alpine
 
   test:
     image: openslides-auth-test
-    command: dockerize -wait tcp://auth:9004 -timeout 10s bash -c "npm run test"
+    command: dockerize -wait tcp://auth:9004 -timeout 30s bash -c "npm run test"
```

Comment:
You can use `wait-for-it` instead of dockerize. You have to create an entrypoint script for this:

```
#!/bin/bash

wait-for-it auth:9004

exec "$@"
```